docs/section12/Rule12-1.md

```diff
@@ -15,24 +15,20 @@
 
 ## Rule Logic:  
 
-- Get the total number of spaces in the building segment in the User model: ```For building_segment in U_RMR.building.building_segments:```  
+- Get the total number of spaces in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
 
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - For each thermal_block in building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```
 
```

Comment:
as pointed out by Charlie in another RDS, this should be 'zone' and not 'thermal_zone'